carbon_black_cloud/plugin.spec.yaml

```diff
@@ -0,0 +1,456 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: carbon_black_cloud
+title: Carbon Black Cloud
+description: The VMware Carbon Black Cloud is a cloud-native endpoint protection platform (EPP) that combines the intelligent system hardening and behavioral prevention needed to keep emerging threats at bay, using a single lightweight agent and an easy-to-use console
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/carbon_black_cloud
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.carbonblack.com/
+tags:
+- carbon black
+- defense
+- antivirus
+- cloud
```

Comment:
```suggestion
- cloud
- vmware
```

Reply:
I committed this change, but it's not showing outdated.